kotsadm/pkg/handlers/logout.go

```diff
@@ -11,22 +11,30 @@ import (
 func Logout(w http.ResponseWriter, r *http.Request) {
 	sess, err := session.Parse(r.Header.Get("Authorization"))
 	if err != nil {
+		if store.GetStore().IsNotFound(err) {
+			JSON(w, http.StatusNoContent, "")
+			return
+		}
+		if r.Header.Get("Authorization") == "" {
+			JSON(w, http.StatusNoContent, "")
+			return
+		}
 		logger.Error(err)
```

Comment:
@divolgin how would you feel about not logging this. It's common when switching clusters, so it's confusing to see this error in logs.

Reply:
if you switch clusters, it should hit the store's IsNotFound